src/core.jl

```diff
@@ -82,16 +132,22 @@ isnull(obj::JavaMetaClass) = obj.ptr == C_NULL
 const JClass = JavaObject{Symbol("java.lang.Class")}
 const JObject = JavaObject{Symbol("java.lang.Object")}
 const JMethod = JavaObject{Symbol("java.lang.reflect.Method")}
+const JConstructor = JavaObject{Symbol("java.lang.reflect.Constructor")}
 const JThread = JavaObject{Symbol("java.lang.Thread")}
 const JClassLoader = JavaObject{Symbol("java.lang.ClassLoader")}
 const JString = JavaObject{Symbol("java.lang.String")}
+const jnull = JavaObject(nothing)
+
+JavaObject(ptr::Ptr{Nothing}) = ptr == C_NULL ? JavaObject(ptr) : JavaObject{Symbol(getclassname(getclass(ptr)))}(ptr)
```

Comment:
This depends on functions in proxy.jl at the moment, so we will probably move it into proxy.jl